contracts/SpokePool.sol

```diff
@@ -341,6 +341,9 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
         uint32 rootBundleId,
         bytes32[] memory proof
     ) public nonReentrant {
+        // This function potentially is expected to transfer WETH to the recipient so we should wrap all ETH.
+        _depositEthToWeth();
```

Comment:
does it make sence to only do this if the `destinationToken` is WETH?

Reply:
sure done